README.md

```diff
@@ -1,15 +1,96 @@
+| [stripe__new_customers_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__churned_customers_monthly.sql)    |  Monthly count of customers that signed up for the first time.                                     |
+| [stripe__platform_fees_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__platform_fees_monthly.sql)    | Monthly fees paid to Stripe.                                                         |
+| [stripe__recovered_customer_revenue_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__recovered_customer_revenue_monthly.sql)    | Monthly revenue due recovery of previously churned customers.               |
+| [stripe__recovered_customers_monthly](https://github.com/HousewareHQ/dbt_stripe_metrics/blob/main/models/metrics/stripe__recovered_customers_monthly.sql)    | Monthly number of previously churned customers who signed up again.|                                                                                                                                 
+
+# 🎯 How do I use the dbt package?
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
+- At least one Fivetran stripe connector syncing data into your destination. 
+- A **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+
+
+## Step 2: Install the package
+
+Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+
+Include in your `packages.yml`
+
+```yaml
+packages:
+  - git: "git@github.com:HousewareHQ/dbt_stripe_metrics.git"
+    revision: 0.1.0
+```
+
+## Step 3: Define database and schema variables
+
+By default, this package will look for your Stripe data in the `stripe` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Stripe data is, please add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+  stripe_source:
+    stripe_database: your_database_name
+    stripe_schema: your_schema_name
+```
+
+For additional configurations for the source models, please visit the [Stripe source package](https://github.com/fivetran/dbt_stripe_source).
+
+## (Optional) Step 4: Change build schema
+By default this package will build the Stripe staging models within a schema titled (<target_schema> + `_stg_stripe`) and the Stripe metrics within a schema titled (<target_schema> + `_stripe_metrics`) in your target database. If this is not where you would like your modeled Stripe data to be written to, add the following configuration to your `dbt_project.yml` file:
```

Comment:
Let's update default target schema to `stg_stripe` in dbt_project.yml

Reply:
or we can remove this configuration to rely on fivetran_source default

Reply:
`stg_stripe` is the default in fivetran `stripe_source` package